extensions/amp-animation/amp-animation.md

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js">&lt;/script></code></td>
```

Comment:
That was supposed to appear as "& l t ;" (without spaces) and not as an open angle brace.